packages/resolve-bus-memory/src/index.js

```diff
@@ -0,0 +1,34 @@
+const eventHandlersMap = new Map();
+
+export default function inMemoryBus() {
+    return {
+        onEvent: (eventTypesArray, callback) => {
+            eventTypesArray.forEach((eventType) => {
+                const callbackArray = eventHandlersMap.get(eventType) || [];
+                callbackArray.push(callback);
+                eventHandlersMap.set(eventType, callbackArray);
+            });
+
+            return () => {
+                eventTypesArray.forEach((eventType) => {
+                    const callbackArray = eventHandlersMap.get(eventType) || [];
```

Comment:
The `|| []` part is redundant - such entry is definitely in map (because of line 9).